sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/DoFnReflectorTest.java

```diff
@@ -533,107 +506,6 @@ public void testBadReturnType() throws Exception {
     DoFnReflector.verifyProcessMethodArguments(getClass().getDeclaredMethod("badReturnType"));
   }
```

Comment:
I like that. I was thinking something along those lines. This is a nice way to manage it.